codegen_x64.c

```diff
@@ -765,8 +765,8 @@ static void emit_func_prologue(Ast *func)
         off -= align(v->ctype->size, 8);
         v->loff = off;
     }
-    for (Iter *i = list_iter(func->localvars); !iter_end(i);) {
-        Ast *v = iter_next(i);
+    for (Iter i = list_iter(func->localvars); !iter_end(i);) {
```

Comment:
By the way, I am thinking of the introduce for [for_each macro](https://lwn.net/Articles/336255/) used in Linux kernel to simplify the similar loops here. Let's do that in another pull request.